### PR TITLE
Add license text

### PR DESCRIPTION
**Pull request by tomhughes**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (C) 2014 Jordan Harband
```
Comment (ljharb) [inline LICENSE]: Thanks - could you change this to 2015, and rebase freshly on top of master?

Comment (tomhughes) [inline LICENSE]: Done.

**Comment by ljharb** [comment]
Please rebase this on top of master, so there won't be any merge commits, and so tests will pass.

**Comment by tomhughes** [comment]
Ah sorry, didn't realise you had just pushed some other commits. Rebased now.

**Comment by ljharb** [comment]
Thanks!

**Pull request merged**